src/plugins/ui/autofill.ts

```diff
@@ -363,6 +368,10 @@ export class AutofillPlugin extends UIPlugin {
     this.autofillZone = { top, bottom, left, right };
   }
 
+  private saveSteps(steps: number) {
```

Comment:
do we really need a one line method for this?
What do you think of inlining the line where the method is used?

Reply:
sure, seems sufficient enough